lib/email/receiver.rb

```diff
@@ -422,6 +426,20 @@ def can_invite?(topic, user)
       !topic.topic_allowed_groups.where("group_id IN (SELECT group_id FROM group_users WHERE user_id = ?)", user.id).exists?
     end
 
+    private
+
+    def check_reply_to_auto_generated_header
+      headers = Mail::Header.new(@mail.body.to_s.gsub("\r\n\r\n", "\r\n")).to_a
```

Comment:
We grab the headers like [how `mail` gem grabs](https://github.com/mikel/mail/blob/master/lib/mail/part.rb#L118) the delivery status report